### DISCUSSION TITLE
Just wanted to say Hi joined today.

### DISCUSSION THREAD

**Discussion by GamesStudioTwo** [axmolengine/axmol#1895] [General]
Hello every body just wanted to say hi. I have decided to join here today as I'm starting to port my stuff to over to axmol witch is going very well at the moment. Cocos is a very good engine but not under development anymore and axmol is really starting to take traction now. I'm seeing a lot of bug fixes witch is great and seem to be the future of the engine as it is extremely fast, lightweight and a quick way of developing for mobile if you willing to put a bit of effete beforehand.  hopefully I will have some demos for you soon or if you need any help or some YouTube videos on programming, tutorials for axmol I would be happy to help you all.

**Comment by halx99** [comment]
welcome

**Comment by thienphuoc** [comment]
welcome

**Comment by aismann** [comment]
>  hopefully I will have some demos for you soon or if you need any help or some YouTube videos on programming, tutorials for axmol I would be happy to help you all.

Thanks. Welcome.

**Comment by johnspeny** [comment]
Super awesome.
